lighting_project/models/product_catalog.py

```diff
@@ -51,19 +26,18 @@ def _compute_project_ids(self):
             )
 
     project_count = fields.Integer(
-        compute="_compute_project_count", string="Projects(s)"
+        compute="_compute_project_count",
+        string="Projects(s)",
     )
 
     def _compute_project_count(self):
-        for record in self:
-            record.project_count = len(record.project_ids)
+        for rec in self:
+            rec.project_count = len(rec.project_ids)
 
     def get_catalog_projects(self):
         self.ensure_one()
-
         action = self.env.ref("lighting_project.product_catalog_action_project").read()[
             0
         ]
```

Comment:
This is deprecated, never use `read()[0]`, from now on use:
```python3
action = self.env["ir.actions.act_window"]._for_xml_id(
            "lighting_project.product_catalog_action_project"
        )
```
If it was a regular action:

```python3
action = self.env["ir.actions.actions"]._for_xml_id(
            "lighting_project.product_catalog_action_project"
        )
``